src/ResourceManagement/Storage/Microsoft.Azure.Management.Storage/generate.cmd

```diff
@@ -14,4 +14,4 @@ set repoRoot=%~dp0..\..\..\..
 set generateFolder=%~dp0Generated
```

Comment:
Note, spec file should be updated to be a snapshot of your spec after your PR is merged, rather than a floating reference to the master branch of the rest-api-specs repo.  In this case, the spec file reference should look like this:

https://raw.githubusercontent.com/Azure/azure-rest-api-specs/9dd2e653a740add998f2c18412a018c4dfd0cde1/arm-storage/2016-12-01/swagger/storage.json